libs/providers/flagd/src/lib/service/in-process/grpc/grpc-fetch.ts

```diff
@@ -27,13 +27,20 @@ export class GrpcFetch implements DataFetch {
   private _isConnected = false;
 
   constructor(config: Config, syncServiceClient?: FlagSyncServiceClient, logger?: Logger) {
-    const { host, port, tls, socketPath, selector } = config;
+    const { host, port, tls, socketPath, selector, defaultAuthority } = config;
+    let clientOptions: ClientOptions | undefined;
```

Comment:
nit: Unless the types are pretty weird, I'd think you could just make this an empty-object const, and then add the `grpc.default_authority` conditionally, which reads better to me.